linux_os/guide/system/auditing/configure_auditd_data_retention/var_auditd_space_left_percentage.var

```diff
@@ -0,0 +1,13 @@
+documentation_complete: true
+
+title: 'The percentage remaining in disk space before prompting space_left_action'
+
+description: 'The setting for space_left as a percentage in /etc/audit/auditd.conf'
+
+type: number
+
+interactive: false
```

Comment:
```suggestion
interactive: true
```

set to true here so in the scap-workbench users can customize to any other value they want. Otherwise it only allows to select the predefined values.